coderd/apikey.go

```diff
@@ -25,6 +25,16 @@ import (
 )
 
 // Creates a new token API key that effectively doesn't expire.
+//
+// @Summary Create token API key
+// @ID create-token-api-key
+// @Security CoderSessionToken
+// @Produce json
+// @Tags Users
+// @Param user path string true "User ID, name, or me"
+// @Param request body codersdk.CreateTokenRequest true "Create token request"
+// @Success 201 {object} codersdk.GenerateAPIKeyResponse
+// @Router /users/{user}/keys/tokens [post]
```

Comment:
I recently learned about `swag fmt`.
```
go run github.com/swaggo/swag/cmd/swag@latest fmt --dir ./coderd
```
Should we enable it for our repo? It's pretty fast.

One gotcha though: Every function needs a proper comment body or go fmt will undo some of the changes.

Example:

`swag fmt`:
```go
//	@Summary	Get audit logs
//	@ID			get-audit-logs
//	@Security	CoderSessionToken
//	@Produce	json
//	@Tags		Audit
//	@Param		q			query		string	true	"Search query"
//	@Param		after_id	query		string	false	"After ID"	format(uuid)
//	@Param		limit		query		int		false	"Page limit"
//	@Param		offset		query		int		false	"Page offset"
//	@Success	200			{object}	codersdk.AuditLogResponse
//	@Router		/audit [get]
```
`go fmt`:
```go
// @Summary	Get audit logs
// @ID			get-audit-logs
// @Security	CoderSessionToken
// @Produce	json
// @Tags		Audit
// @Param		q			query		string	true	"Search query"
// @Param		after_id	query		string	false	"After ID"	format(uuid)
// @Param		limit		query		int		false	"Page limit"
// @Param		offset		query		int		false	"Page offset"
// @Success	200			{object}	codersdk.AuditLogResponse
// @Router		/audit [get]
```

(Notice tab prefix -> space.)

However, with a comment it's fine.
```go
// Get audit logs.
//
//	@Summary	Get audit logs
//	@ID			get-audit-logs
//	@Security	CoderSessionToken
//	@Produce	json
//	@Tags		Audit
//	@Param		q			query		string	true	"Search query"
//	@Param		after_id	query		string	false	"After ID"	format(uuid)
//	@Param		limit		query		int		false	"Page limit"
//	@Param		offset		query		int		false	"Page offset"
//	@Success	200			{object}	codersdk.AuditLogResponse
//	@Router		/audit [get]
```

I dislike the full-on tab indentation though (will be messy for different settings of editor indentation display). We could submit a PR to `swaggo/swag` to use space indentation for everything _except_ the prefix. Related PR that changed the behavior: https://github.com/swaggo/swag/pull/1386 from all space to all tab (vs just fixing the prefix).

Reply:
I can enable it in a separate PR to cover all change, but I'm concerned if it doesn't conflict with the project formatter.